config/testdata/http.conf.bearer-token-and-file-set.bad.yml

```diff
@@ -0,0 +1,16 @@
+basic_auth:
+  username: username
+  password: "mysecret"
+bearer_token: mysecret
+bearer_token_file: file
+proxy_url:
```

Comment:
This still doesn't look right. You need to copy the unmarshaller and marshaller over.